File: EDD-master/EDD-Unitat-4/MAKE/calculadora.c
#include "MEUCALC.h"
#include <stdio.h>
int main () {

int op1,op2,s,r,m,d,ma;
printf("Introdueix L'operador 1\n");
scanf("%d",&op1);
printf("Introdueix L'operador 2\n");
scanf("%d",&op2);
s=suma(op1,op2);
r=resta(op1,op2);
m=multiplica(op1,op2);
d=divideix(op1,op2);
ma=major(op1,op2);

if (ma == 1) { ma = op1; }
else {ma = op2;}

printf("La suma es %d\nLa resta es %d\nLa multiplicacio es %d\nLa divisio es %d\nEl nombre major es%d\n",s,r,m,d,ma);

}

File: EDD-Unitat-4/JAVA/gradle/gradleproject1/src/main/java/gradleproject1/Calculadora.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gradleproject1;

public class Calculadora {

    private float lastResult;
    private String lastOp;

    public float getLastResult(){
        return this.lastResult;
    }

    public String getLastOp(){
        return this.lastOp;
    }

    public float suma(float op1, float op2){
        float result=op1+op2;
        this.lastResult=result;
        this.lastOp="Suma";

        return result;
    }

    public float resta(float op1, float op2){
        float result=op1-op2;
        this.lastResult=result;
        this.lastOp="Resta";

        return result;
    }

    public float multiplica(float op1, float op2){
        // Fem els càlculs
        float result=op1*op2;

        // Actualitzem els atributs de la classe
        this.lastResult=result;
        this.lastOp="Multiplica";

        // I finalment retornem els resultats
        return result;
    }

    public float divideix(float op1, float op2){
        // Fem els càlculs
        float result=op1/op2;

        // Actualitzem els atributs de la classe
        this.lastResult=result;
        this.lastOp="Divideix";

        // I finalment retornem els resultats
        return result;
    }
    public boolean major(float op1, float op2){
        // Fem els càlculs
        boolean resultat=op1>op2;
        return resultat;
    }
   public float mitjana(float op1, float op2){
        // Fem els càlculs
        float resultat=(op1+op2)/2;
        return resultat;
    }

}
